Makefile

```diff
@@ -54,3 +53,15 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+rpm:
+	mkdir -p dist
+	chmod 777 `pwd`/dist
+	cd packages && docker build -t calico-mesos-rpm-builder .
```

Comment:
Instead of cd'ing and then using the current directory, just tell it to use that directory:
```
docker build -t calico-mesos-rpm-builder ./packages
```